src/utils/scripts.ts

```diff
@@ -1,72 +1,108 @@
-import { spawn } from "child_process";
+/* eslint-disable no-console */
+import { execSync, spawn } from "child_process";
 import { UserError } from "../errors.js";
 import colors from "colors";
 import _ from "lodash";
 import { Logger } from "tslog";
-import { execaCommand } from "execa";
-import { debugLogger, log } from "./logging.js";
+import { FunctionConfiguration } from "../models/projectConfiguration.js";
+import { YamlProjectConfiguration } from "../projectConfiguration/yaml/v2.js";
+import { FunctionType } from "../projectConfiguration/yaml/models.js";
+import getProjectInfoByName from "../requests/getProjectInfoByName.js";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function isFunctionConfiguration(value: any): value is FunctionConfiguration {
```

Comment:
You should use unknown instead of any. This way no warning is raised.
```suggestion
function isFunctionConfiguration(value: unknown): value is FunctionConfiguration {
```